buildbot-conf/cordova-config.json.sample

```diff
@@ -12,8 +12,8 @@
     "appium": {
         "androidDeviceName":      "Galaxy_Nexus",
         "androidPlatformVersion": "19",
-        "iosDeviceName":          "iPhone 5",
-        "iosPlatformVersion":     "8.4",
+        "iosDeviceName":          "iPhone 6s",
+        "iosPlatformVersion":     "9.1",
         "timeout":                2400,
```

Comment:
Just a friendly reminder to make this change in the actual file on the Buildbot server; this file is just a sample.